cwms-data-api/src/test/java/cwms/cda/data/dto/basin/BasinTest.java

```diff
@@ -121,10 +128,9 @@ private Basin buildTestBasin(CwmsId basinId, boolean parentBasinIdNull) {
         }
     }
 
-    private static void assertSame(Basin first, Basin second){
+    public static void assertSame(Basin first, Basin second){
         assertAll(
-                () -> assertEquals(first.getBasinId().getName(), second.getBasinId().getName(), "Basin IDs are not equal"),
-                () -> assertEquals(first.getBasinId().getOfficeId(), second.getBasinId().getOfficeId(), "Basin Office IDs are not equal"),
+                () -> CwmsIdTest.assertSame(first.getBasinId(), second.getBasinId()),
                 () -> assertEquals(first.getPrimaryStreamId().getName(), second.getPrimaryStreamId().getName(), "Primary Stream IDs are not equal"),
```

Comment:
This should also use CwmsIdTest to compare the primary stream ids

Reply:
Addressed in commit